src/js/views/patients/shared/list.scss

```diff
@@ -3,7 +3,6 @@
   align-items: center;
   color: color(light_blue);
   display: flex;
-  line-height: 1;
```

Comment:
Removing this so the browser uses `line-height: normal` fixes the issue. And I couldn't find any negative side effects because of it.

Reply:
I like removing it, but it was added here: https://github.com/RoundingWell/care-ops-frontend/pull/837. Sure this is not breaking something elsewhere?